src/AppBundle/Resources/views/Org/ClientProfile/_detailsBanner.html.twig

```diff
@@ -29,7 +35,6 @@
             {% endif %}
         </h3>
     </div>
-{% if app.user and (app.user.isDeputyProf() or app.user.isDeputyPa()) and report is defined and report is not null
-   and report.client is not null %}
+{% if showDetailsBanner %}
```

Comment:
It seems like we don't want this line, and instead the closing div below should be placed before the `{% endif %}` on line 20?

Reply:
`showDetailsBanner` is controlling a surrounding div. So, if false it looks like:
```twig
<p>Stuff inside</p>
```
and if true:
```twig
<div class="...">
  <p>Stuff inside</p>
</div>
```

So we need to wrap the starting tag and the ending tag of the surrounding div in two different if statements:

```twig
{% if showDetailsBanner %}<div class="...">{% endif %}
  <p>Stuff inside</p>
{% if showDetailsBanner %}</div>{% endif %}
```

Reply:
Okay ,that makes sense cheers